src/setup.ts

```diff
@@ -39,8 +40,45 @@ async function setup() {
   }
   core.info(`try to setup sccache version: ${version}`);
 
+  // Search local file system cache for sccache.
+  // This is useful when actions run on a self-hosted runner.
+  let sccacheHome = find('sccache', version);
+  if (sccacheHome === '') {
```

Comment:
It seems `find` returns empty string when a tool is not found.

https://github.com/actions/toolkit/blob/361a115e538ac6d8eb06cc47f3fcecce557d04c8/packages/tool-cache/src/tool-cache.ts#L518